### ISSUE TITLE
CameraSender does not request camera permission

### ISSUE THREAD

**Issue by snooplsm** [layerhq/Atlas-Android#64]
App will crash if you try and use the camerasender on marshmallow because it doesn't have the camera permission.  GallerySender handles the permission check though.

**Comment by cpiggott** [comment]
I noticed that the Camera permission was also removed in the Atlas-Android-Messenger [AndroidManifest.xml a few commits ago](https://github.com/layerhq/Atlas-Android-Messenger/commit/cbe742faf2adf42960cc8fa925dec3e2441f4539). I'm guessing since the `ACTION_IMAGE_CAPTURE` doesn't require the` CAMERA` permission, there isn't a need to request for the permission.

At first I thought it might be the `getContext().getExternalFilesDir(android.os.Environment.DIRECTORY_PICTURES)` call not having write permission, but after reading through the documentation, it appears that this doesn't require `.WRITE_EXTERNAL_STORAGE` as of [KitKat.](http://developer.android.com/reference/android/content/Context.html#getExternalFilesDir(java.lang.String))
 
I know the way permissions are granted has changed for Marshmallow, but unless they forgot to update the docs, it shouldn't be an issue.
